CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 4.4.2
+- Fix changelog path
+- Fix stack trace capture
```

Comment:
I'd move this down the list. Can we ad a link to the PR so people can dig in more if they want to? Maybe just the short commit id? Seems like this is too few words to be helpful.